src/util.c

```diff
@@ -107,4 +135,32 @@ long get_directory_size(const char *dir_path)
         return total_size;
 }
 
+/**
+ * Generates a 32 characters random id
+ *
+ * @param buffer pointer to allocate and copy data
+ *
+ */
+int get_random_id(char **buff)
+{
```

Comment:
There is already a function to generate a random machine ID (`generate_machine_id()` in src/telemdaemon.c). Can we consolidate this function with that one?